notebooks/passenger_vehicle_co2_lca.py

```diff
@@ -0,0 +1,322 @@
+"""Passenger Vehicle CO2 LCA."""
+
+# ---
+# jupyter:
+#   jupytext:
+#     text_representation:
+#       extension: .py
+#       format_name: light
+#       format_version: '1.5'
+#       jupytext_version: 1.14.5
+#   kernelspec:
+#     display_name: Python 3 (ipykernel)
+#     language: python
+#     name: python3
+# ---
+
+# # $CO_2$-LCA för olika bilalternativ
+#
+# **Human activities, principally through emissions of greenhouse gases, have unequivocally
+# caused global warming, with global surface temperature reaching 1.1°C above 1850-1900
+# in 2011-2020. Global greenhouse gas emissions have continued to increase, with unequal
+# historical and ongoing contributions arising from unsustainable energy use, land use and
+# land-use change, lifestyles and patterns of consumption and production across regions,
+# between and within countries, and among individuals (high confidence)**
+# https://www.ipcc.ch/report/ar6/syr/downloads/report/IPCC_AR6_SYR_SPM.pdf
+#
+# Ett steg mot konkreta och effektiva åtgärder är att göra en livscykelanalys för fordon. Nedan en enkel jämförelsemodell för att kunna utvärdera utsläpp ifrån nyproduktion i förhållande till utsläpp från körning. De exempel på siffror som tagits fram kommer från öppna källor på nätet, t ex Volvo Cars LCA-rapporter, men lämnas för användaren att ändra för att kunna göra egna parameterstudier.
+
+import matplotlib.pyplot as plt
+import numpy as np
+
+# ## Grundläggande antaganden
+# Grundläggande antaganden kommer från Volvo Cars LCA-rapporter, som återfinns här:
+#
+# https://www.volvocars.com/mt/news/sustainability/transparency-in-action-heres-our-ex90-carbon-footprint-report/
+#
+# Siffrorna för CO2 per liter bensin kan justeras uppåt något ytterligare enligt denna källa:
+#
+# https://innovationorigins.com/en/producing-gasoline-and-diesel-emits-more-co2-than-we-thought/
+#
+# I denna analys har dock datan från Volvo använts.
+
+# Grundläggande antaganden
+car_manufacturing_cost = 8  # 8 kg CO2/kg car
+battery_manufacturing_cost = 77  # 77 kg CO2/kWh battery (average between 56 & 98, from EX90 and EX30, respectively.)
+kgC02_per_kWh_europe = 0.45  # 0.45 kg CO2/kWh electricity (World mix)
+kgC02_per_litre = 3  # 3kg C02/litre (extraction, production & combustion)
```

Comment:
per litre what? Petrol?

Reply:
Petrol, yes. Should be clarified, good point!